quill-jasync-zio/src/main/scala/io/getquill/context/zio/ZIOMonad.scala

```diff
@@ -0,0 +1,57 @@
+package io.getquill.context.zio
```

Comment:
This is really cool but I think I'm going to deprecate Quill's IO monad.

Reply:
that's ok :)